### PR TITLE
Translate README for chinese

### PR DESCRIPTION
**Pull request by ccworld1000**
Translate README for chinese

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ There is also a command line tool called "p6doc".
 (If you are browsing this repository via GitHub, it will not display most
 files correctly, because this is Perl 6 Pod, and GitHub assumes Perl 5 POD).
 
+## Translate README
```
Comment (JJ) [inline README.md]: Please change this to "README translations" or "README in other languages"

Comment (JJ) [inline README.md]: Maybe also add "README in Chinese" to clarify.

Comment (ccworld1000) [reply README.md]: OK, I've changed it again!

Comment (ccworld1000) [reply README.md]: OK, I've changed it again!

Comment (ccworld1000) [reply README.md]: OK, I've changed it again!

**Pull request merged**